apps/expo/src/app/muscleGroup/index.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useState } from 'react'
+import { Dimensions, Image, Text, View, StyleSheet } from 'react-native'
+import { router } from 'expo-router'
+import tailwind from '@/tooling/tailwind'
+import { Ionicons, MaterialCommunityIcons } from '@expo/vector-icons'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import Body from 'react-native-body-highlighter'
+import ToggleSwitch from '~/components/toggle/Toggle'
+import FrontBackSwitch from '~/components/muscleGroup/FrontBackSwitch'
+
+export type muscleSelect =
+  | "abs"
+  | "adductors"
+  | "ankles"
+  | "biceps"
+  | "calves"
+  | "chest"
+  | "deltoids"
+  | "deltoids"
+  | "feet"
+  | "forearm"
+  | "gluteal"
+  | "hamstring"
+  | "hands"
+  | "hair"
+  | "head"
+  | "knees"
+  | "lower-back"
+  | "neck"
+  | "obliques"
+  | "quadriceps"
+  | "tibialis"
+  | "trapezius"
+  | "triceps"
+  | "upper-back";
+
+export default function NewWorkoutMuscleGroup() {
+  const { width: screenWidth, height: screenHeight } = Dimensions.get('window')
+  const vmin70 = Math.min(screenWidth, screenHeight) * 0.7
+
+  const [bodyPartSelected, setBodyPartSelected] = useState({/*
+    slug: "biceps",
+    intensity: 2,*/
+  }
+  )
+  const [isBackSideEnabled, setIsBackSideEnabled] = useState(false)
+  const [isMale, setIsMale] = useState(true)
+  const toggleSwitch = () =>
+    setIsBackSideEnabled((previousState) => !previousState)
+
+  const toggleGenderSwitch = () => setIsMale((previousState) => !previousState)
+
+  //let tmp = "";
+  let sameGroup = {};
+  const [tmp1, setTmp1] = useState({})
+  const [userSelection, setUserSelection] = useState("");
+
```

Comment:
const handleMusclePress = (e) => { if(e == userSelection){ setUserSelection("") }}

Reply:
I'll add additional selection functionality to the <Body> in next PR